src/browser/api/window.js

```diff
@@ -450,12 +449,6 @@ Window.create = function(id, opts) {
             ofEvents.emit(route.window('frame-disconnected', uuid, name), payload);
         };
 
-        // this is a first pass at teardown. for now, push the unsubscribe
-        // function for each subscription you make, on closed, remove them all
-        // if you listen on 'closed' it will crash as your resources are
```

Comment:
At BrowserWindow's "closed" event only its WebContents are gone, at least in today's OF world